addons/mail/static/src/components/message/message.xml

```diff
@@ -162,9 +162,9 @@
                             </t>
                         </div>
                         <AttachmentList t-if="messageView.attachmentList" attachmentListLocalId="messageView.attachmentList.localId"/>
-                        <div t-if="messageView.message.messageReactionGroups.length > 0" class="d-flex flex-wrap ml-2">
-                            <t t-foreach="messageView.message.messageReactionGroups" t-as="messageReactionGroup" t-key="messageReactionGroup.localId">
-                                <MessageReactionGroup className="'mr-1 mb-1'" messageReactionGroupLocalId="messageReactionGroup.localId"/>
+                        <div t-if="messageView.messageReactionGroupViews.length > 0" class="d-flex flex-wrap ml-2">
+                            <t t-foreach="messageView.messageReactionGroupViews" t-as="messageReactionGroupView" t-key="messageReactionGroupView.localId">
+                                <MessageReactionGroup class="mr-1 mb-1" messageReactionGroupViewLocalId="messageReactionGroupView.localId"/>
```

Comment:
`class="mr-1 mb-1"` => `className="'mr-1 mb-1'"`

(`class` still works, but OWL 2 will kill this so prefer `className`)